spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/building.adoc

```diff
@@ -0,0 +1,85 @@
+[[building]]
+== Building
+
+:jdkversion: 1.7
+
+=== Basic Compile and Test
+
+To build the source you will need to install JDK {jdkversion}.
+
+The build uses the Maven wrapper so you don't have to install a specific
+version of Maven.  To enable the tests for Redis you should run the server
+before bulding.  See below for more information on how run Redis.
+
+The main build command is
+
+----
+$ ./mvnw clean install
+----
+
+You can also add '-DskipTests' if you like, to avoid running the tests.
+
+NOTE: You can also install Maven (>=3.3.3) yourself and run the `mvn` command
+in place of `./mvnw` in the examples below. If you do that you also
+might need to add `-P spring` if your local Maven settings do not
+contain repository declarations for spring pre-release artifacts.
+
+NOTE: Be aware that you might need to increase the amount of memory
+available to Maven by setting a `MAVEN_OPTS` environment variable with
+a value like `-Xmx512m -XX:MaxPermSize=128m`. We try to cover this in
+the `.mvn` configuration, so if you find you have to do it to make a
+build succeed, please raise a ticket to get the settings added to
+source control.
+
+
+The projects that require middleware generally include a
+`docker-compose.yml`, so consider using
+http://compose.docker.io/[Docker Compose] to run the middeware servers
```

Comment:
I guess so. I don't even understand why we talk about docker compose here. Again, this is copy/paste from Spring Cloud. All projects have those (broken) links